spec/granite/migrator/migrator_spec.cr

```diff
@@ -0,0 +1,135 @@
+require "../../spec_helper"
+
+describe Granite::Migrator::Base do
+  describe "#drop_sql" do
+    it "generates correct SQL with #{{{ env("CURRENT_ADAPTER") }}} adapter" do
+      {% if env("CURRENT_ADAPTER") == "mysql" %}
+        Review.migrator.drop_sql.should eq "DROP TABLE IF EXISTS `reviews`;"
+      {% else %}
+        Review.migrator.drop_sql.should eq "DROP TABLE IF EXISTS \"reviews\";"
+      {% end %}
+    end
+  end
+
+  describe "#create_sql" do
+    it "generates correct SQL with #{{{ env("CURRENT_ADAPTER") }}} adapter" do
+      {% if env("CURRENT_ADAPTER") == "pg" %}
+        Review.migrator.create_sql.should eq <<-SQL
+          CREATE TABLE "reviews"(
+          "id" BIGSERIAL PRIMARY KEY,
+          "name" VARCHAR(255)
```

Comment:
Should probably make `name` be `TEXT`, seems to be the preferred variable length string type for PG.